torch/csrc/distributed/rpc/tensorpipe_agent.cpp

```diff
@@ -266,7 +266,7 @@ std::unique_ptr<ChannelRegistration> makeMultiplexedUvChannel() {
   std::vector<std::shared_ptr<tensorpipe::transport::Listener>> listeners;
```

Comment:
I can't suggest fixing the previous line but due to Github limitations, but:
```suggestion
  std::vector<std::shared_ptr<tensorpipe::transport::Context>> contexts;
  contexts.reserve(kNumUvThreads);
  std::vector<std::shared_ptr<tensorpipe::transport::Listener>> listeners;
  listeners.reserve(kNumUvThreads);
```